cloud_utils/vault.py

```diff
@@ -6,6 +6,10 @@
 base_metadata_url = "http://169.254.169.254/metadata"
 
 
+class KeyDoesNotExist(Exception):
```

Comment:
private ?

Reply:
It will be needed when using read_key outside of the lib